### PR TITLE
Add Spring web operation name handler

### PR DESCRIPTION
**Pull request by zouyx**
1.Support Spring RequestMapping
2.HandlerUtils.requestMapping，if use ```.path()```can not get anything when use ```@Requestmapping```
so change to value()

### CODE REVIEW

File: opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/HandlerInterceptorSpanDecorator.java [hunk 1]
```diff
@@ -132,6 +132,34 @@ public void onAfterConcurrentHandlingStarted(HttpServletRequest httpServletReque
         }
     };
 
+    /**
+     * Decorator to record details about the handler as log events recorded on the span.
```
Comment (pavolloffay) [inline opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/HandlerInterceptorSpanDecorator.java]: It does not seem to set any logs on the span.

Comment (zouyx) [reply opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/HandlerInterceptorSpanDecorator.java]: thanks, fixed it

File: opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/HandlerInterceptorSpanDecorator.java [hunk 2]
```diff
@@ -132,6 +132,33 @@ public void onAfterConcurrentHandlingStarted(HttpServletRequest httpServletReque
         }
     };
 
+    /**
+     * Use the Spring RequestMapping's value as the span's operation name.
+     */
+    HandlerInterceptorSpanDecorator HANDLER_SPRING_WEB_METHOD_OPERATION_NAME = new HandlerInterceptorSpanDecorator() {
```
Comment (pavolloffay) [inline opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/HandlerInterceptorSpanDecorator.java:138]: Could you please rename this to `REQUEST_MAPPING_OPERATION_NAME`?

**Comment by thejasgit** [comment]
any update on this?

**Comment by zouyx** [comment]
> any update on this?

Do you want to use this feature?

**Comment by thejasgit** [comment]
> > any update on this?
> 
> Do you want to use this feature?

Yes, Currently I just copied your code in my application. It would be nice if this is made available as part of the library.

**Comment by zouyx** [comment]
> > > any update on this?
> > 
> > 
> > Do you want to use this feature?
> 
> Yes, Currently I just copied your code in my application. It would be nice if this is made available as part of the library.

Ok, I tried to finish this feature in this week. @pavolloffay

**Pull request closed without merging**